src/stories/Blocks/advanced-search/advanced-search.scss

```diff
@@ -0,0 +1,152 @@
+.advanced-search {
+  width: 100%;
+  padding: 40px $s-md;
```

Comment:
What is you thoughts about NOT using spacing variables?

Reply:
Well we don't have a 40px one. I think I tried to match them up if they were defined, but just opted for the values in case they weren't.

Reply:
I have no strong opinion about this, other than the spacing system has failed 😃 
We can talk about if it should have an overhaul.
One idea could be to consider the existing system to be deprecated and keep it, and use something new onwards.